### DISCUSSION TITLE
react-p5-wrapper in React Native

### DISCUSSION THREAD

**Discussion by Terceramayor** [P5-wrapper/react#124] [General]
Nice work on this library. I was wondering whether it can be used on React Native, is it compatible? As far as I know it relies in DOM elements, such as canvas, which are not present in React Native.

**Comment by jamesrweb** [comment]
Right now this library is built for React in the browser. I have been considering ports for Angular, Vue and Elm lately due to user feedback but I will add React Native to the list to validate the possibility 💭. Right now though I don't know enough to say if it is possible or not since I haven't used much React Native before though.

**Reply by Terceramayor** [reply]
Nice, thanks for your positive reaction.
